tests/integration_tests/test_experiment.py

```diff
@@ -348,12 +351,13 @@ def test_experiment_image_inputs(image_parms: ImageParms, csv_filename: str):
 @pytest.mark.parametrize('train_format', IMAGE_DATA_FORMATS_TO_TEST)
 def test_experiment_image_dataset(
         train_format, train_in_memory,
-        test_format, test_in_memory
+        test_format, test_in_memory,
+        tmpdir
 ):
     # primary focus of this test is to determine if exceptions are
     # raised for different data set formats and in_memory setting
     # Image Inputs
-    image_dest_folder = os.path.join(os.getcwd(), 'generated_images')
+    image_dest_folder = os.path.join(os.getcwd(),tmpdir+'generated_images')
```

Comment:
combining `os.getwd()` and `tmpdir` to form may not be the correct approach.

Reply:
Yeah, I would ditch the cwd stuff and consider creating a new tempdir to store the images:

```
with tempfile.TemporaryDirectory() as image_dest_folder:
    ...
```